<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewBibolamazifileDialog</class>
 <widget class="QDialog" name="NewBibolamazifileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create new bibolamazi file</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>So you'd like to organize some bibtex entries...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stk">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="srcmulti">
         <property name="title">
          <string>Would you like to collect entries from several bibtex files?</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0" colspan="2">
           <widget class="QRadioButton" name="rdbtnMergeMultiple">
            <property name="text">
             <string>I would like to merge bibliography entries from different files</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QRadioButton" name="rdbtnSingleSource">
            <property name="text">
             <string>My bibliography entries are currently all in one bibtex file</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Specify sources where to collect bibtex entries from</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTextBrowser" name="txtSources">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QPushButton" name="btnSrcSet">
              <property name="text">
               <string>set</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSrcAdd">
              <property name="text">
               <string>add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSrcClear">
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="chkDuplicatesFilter">
         <property name="title">
          <string>Merge duplicate entries with different key names &amp;&amp; create aliases</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Citing the same article using different keys, such as &lt;code&gt;\cite{Einstein1937}&lt;/code&gt; and &lt;code&gt;\cite{EPRpaper}&lt;/code&gt; will refer to the same bibliography entry.&lt;/p&gt;
&lt;p style=&quot;font-style:italic&quot;&gt;In order for this to work, you need to add the following line to the preamble of your document:&lt;/p&gt;&lt;pre&gt;\input{bibolamazi_dup_aliases.tex}&lt;/pre&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="chkArxivUnpublished">
         <property name="title">
          <string>Normalize arxiv fields in entries, for *unpublished* entries</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0">
           <widget class="QRadioButton" name="rdbtnArxivUnpub_unpubnote">
            <property name="text">
             <string>Set entry type to @unpublished and add &quot;note = {arXiv:XXXX.ZZZZZ}&quot;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rdbtnArxivUnpub_eprint">
            <property name="text">
             <string>Use &quot;eprint&quot; field, like &quot;eprint = {XXXX.ZZZZZ}&quot;</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="chkArxivUnpubIncludeTheses">
            <property name="text">
             <string>Include theses as unpublished entries</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="chkArxivPublished">
         <property name="title">
          <string>Normalize arxiv fields in entries, for *published* entries</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QRadioButton" name="rdbtnArxivPub_eprint">
            <property name="text">
             <string>Use &quot;eprint&quot; field, like &quot;eprint = {XXXX.ZZZZZ}&quot;</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rdbtnArxivPub_unpubnote">
            <property name="text">
             <string>Add &quot;note = {arXiv:XXXX.ZZZZZ}&quot;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rdbtnArxivPub_strip">
            <property name="text">
             <string>Strip arxiv information entirely from published entries</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkArxivPubIncludeTheses">
            <property name="text">
             <string>Include theses as published entries</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="chkUrlFilter">
         <property name="title">
          <string>Fix URLs</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QCheckBox" name="chkUrlStrip">
            <property name="text">
             <string>Remove all URLs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkUrlStripAllIfDoiOrArxiv">
            <property name="text">
             <string>Remove all URLs if there is an arXiv id or a DOI</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkUrlStripDoiArxiv">
            <property name="text">
             <string>Remove URLs that simply refer to a DOI or to an arxiv page</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkUrlKeepFirstUrlOnly">
            <property name="text">
             <string>Keep only the first URL available</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="chkFixesFilter">
         <property name="title">
          <string>Apply general fixes</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="chkFixesGeneralSelection">
            <property name="text">
             <string>Include a selection of general fixes</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkFixesEncodeToLatex">
            <property name="text">
             <string>Encode accents and special characters into LaTeX</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="chkKeepOnlyUsed">
         <property name="title">
          <string>Keep only entries that I actually use in my LaTeX document</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;body&gt;&lt;p style=&quot;font-style:italic&quot;&gt;Make sure when you save the bibolamazi file to give it the same base name as your LaTeX document. For instance, if your document is named &lt;code&gt;example.tex&lt;/code&gt;, save the bibolamazifile as &lt;code&gt;example.bibolamazi.bib&lt;/code&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnBack">
       <property name="text">
        <string>◀︎ back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNext">
       <property name="text">
        <string>next ▶</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveFinish">
       <property name="text">
        <string>save file ▶</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rdbtnSingleSource</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnSrcSet</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdbtnMergeMultiple</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnSrcAdd</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdbtnMergeMultiple</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnSrcClear</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdbtnMergeMultiple</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkDuplicatesFilter</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>NewBibolamazifileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUrlStrip</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkUrlStripAllIfDoiOrArxiv</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUrlStrip</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkUrlKeepFirstUrlOnly</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUrlStrip</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkUrlStripDoiArxiv</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
